internal/cmd/config/set/set.go

```diff
@@ -23,6 +23,7 @@ const (
 	serviceAccountCustomEndpointFlag  = "service-account-custom-endpoint"
 	skeCustomEndpointFlag             = "ske-custom-endpoint"
 	resourceManagerCustomEndpointFlag = "resource-manager-custom-endpoint"
+	OpenSearchCustomEndpointFlag      = "opensearch-custom-endpoint"
```

Comment:
```suggestion
	openSearchCustomEndpointFlag      = "opensearch-custom-endpoint"
```
Unless there's a reason for this to be public